src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
Keep your attributes correctly formatted. If the HTML element has long attribute values or a number of attributes is more than 2, start each one on a new line with 2-space indentation relative to the tag. The <meta> tag content attribute value is too long and should be broken down into multiple lines.